modules/client-configuration/pages/clients-sleses.adoc

```diff
@@ -215,7 +215,7 @@ spacewalk-repo-sync -c channel_name -u https://EXAMPLE.COM/pub/rhel/AppStream/
 At the command prompt as root enter the following:
 +
 ----
-spacewalk-repo-sync -c channel_name -u https://EXAMPLE.COM/pub/<os_name>/
+spacewalk-repo-sync -c channel_name -u https://<SERVER-FQDN>/pub/<os_name>/
```

Comment:
```suggestion
spacewalk-repo-sync -c channel_name -u https://susemanager.example.com/pub/<os_name>/
```

Reply:
Thank you for the suggestions.
It turned out that the command line part discussed here was dropped in PR #151 (*).  Therefore, I am going to address just the existent part in the code base, i.e.,
- Use <os_name> in the examples
- Refer to sleses instead of centOS/RHEL.

(*) In case we wish to reinclude the command line instructions, then I suggest we open a new issue to address that.